src/public/lib/nbgv-python/src/nbgv_python/templating.py

```diff
@@ -166,7 +166,7 @@ def _format_component(
     double_quote: bool,
 ) -> str:
     if isinstance(value, bool):
-        return _quote(str(value).lower(), double_quote=double_quote)
+        return "True" if value else "False"
```

Comment:
Good catch handling bool before int (since bool is a subclass of int). This ensures version_tuple renders valid Python literals instead of 1/0 and matches the documented breaking change.